state-estimation-localization-slam/README.md

```diff
@@ -2,42 +2,47 @@
 
 Use noisy inputs from multiple sensors, to estimate, as accurately as possible, the current location of the robot within its environment, and also, in some cases, build a map of its surroundings as it explores the area.
 
-**Prerequisites**- Basic Knowledge about Probability, Jacobians, Linear Algebra and Numpy in python.
+## Prerequisites
 
-**Course is fit for-** This field being a part of Probabilistic Robotics is fit for people who are interested in the planning, mobile robotics, software aspect of robotics and developing autonomous robots.Navigation is one of the most challenging competencies required of a mobile robot. Success in navigation requires success at the four building blocks of navigation:
+Basic Knowledge about Probability, Jacobians, Linear Algebra and Numpy in python.
 
-**Perception**- the robot must interpret its sensors to extract meaningful data 
 
-**Localization**- the robot must determine its position in the environment
+## Course is fit for
 
-**Cognition**- the robot must decide how to act to achieve its goals
+This field being a part of Probabilistic Robotics is fit for people who are interested in the planning, mobile robotics, software aspect of robotics and developing autonomous robots.Navigation is one of the most challenging competencies required of a mobile robot. Success in navigation requires success at the four building blocks of navigation:
```

Comment:
"... developing autonomous robots.Navigation is one" 
add space between robots.Navigation